dqgen/services/__init__.py

```diff
@@ -83,4 +83,4 @@
                                        "statistics.html": HTML_TEMPLATES.get_template("statistics.jinja2")}
 
 # QUERY_FALLBACK_LANGUAGES = ["en", "fr", "de", "es"]
-QUERY_FALLBACK_LANGUAGES = ["en"]
\ No newline at end of file
+QUERY_FALLBACK_LANGUAGES = ["en", "fr"]
```

Comment:
why 2? 
in one variant we shall have "en" and in the second variant the four above